### PR TITLE
Update openSUSE install description

### PR DESCRIPTION
**Pull request by ghost**
Leap 42.x is not relevant anymore, and Leap 15.0 and
Leap 15.1 have flatpak in the default repo.

### CODE REVIEW

File: data/distro.yml
Comment (Conan-Kudo) [inline data/distro.yml]: It probably makes sense to drop this notice entirely, since the warning says that everything works fine by default in all supported versions of openSUSE.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]

File: data/distro.yml
Comment (sknorr) [reply data/distro.yml]: True. Will fix.

Comment (allanday) [reply data/distro.yml]: If Flatpak is already installed, it would be better to remove the installation instructions altogether. That's what the pages for other distros do.

Comment (sknorr) [reply data/distro.yml]: I don't think it is, however. Certainly not on all installation options (i.e. KDE, GNOME, minimal server, etc.).
The note was about that however, apparently on some openSUSE version you had to use GNOME Software rather than the flatpak command line tool..? Tbh, I never experienced that even on older versions of openSUSE.

**Comment by sknorr** [comment]
Hello -- can this be merged now?

File: data/distro.yml
Comment (AsciiWolf) [reply data/distro.yml]: Is GNOME Software installed by default on the GNOME variant of openSUSE? If so, does the 1-click installer of _flatpak_ package also activate its support in GNOME Software so Flatpak applications can be installed using the GNOME Software GUI?

Comment (ghost) [reply data/distro.yml]: GNOME Software is [included in the GNOME pattern](https://build.opensuse.org/package/view_file/openSUSE:Leap:15.1/patterns-gnome/patterns-gnome.spec?expand=1), so yes, Software will be installed by default with the GNOME variant of openSUSE. However, a large part of the openSUSE population uses their OS with KDE.

The `gnome-software` package includes `libgs_plugin_flatpak.so` and requires the `libflatpak` package, so Flatpak support in GNOME Software is a given.
The `flatpak` command-line tool needs to be installed manually.

The "1-Click" installer for `flatpak` installs the command-line tool and the libflatpak library. It has nothing to do with GNOME Software. If you want to install GNOME software using the "1-Click" thing, you need a separate ymp file.

[Also, note that I am quoting "1-Click" for a reason -- it's more like a ten-click installer.]

Comment (AsciiWolf) [reply data/distro.yml]: So, what about KDE Discover? Is it installed by default with the Flatpak support included on the KDE variant of openSUSE?

**Comment by AsciiWolf** [comment]
@sknorr LGTM, if there is a Flatpak support in GNOME Software/KDE Discover.

File: data/distro.yml
Comment (ghost) [reply data/distro.yml]: The KDE pattern only recommends `discover` and `discover` in turn recommends its Flatpak back-end. This means it very much depends on your system settings whether it will be installed (there is a Zypper setting that determines whether to install recommended packages or not, I think it is set to not install those by default now).

However, openSUSE ships a myriad other desktops as well and if I go through all of them, I will never finish.

I also don't understand where these questions are leading. I just wanted to make a simple PR to remove the mentions of issues with completely unsupported OS versions.

**Comment by AsciiWolf** [comment]
Why was this closed?

**Comment by ghost** [comment]
Sorry, I was a bit down about the state of this minor PR after close to 3 months. :/ That was a bit unfair. I can re-create the PR if you want.

**Comment by barthalion** [comment]
Sorry @sknorr, we're doing our best. Could you bring it back? I'll look into bringing more people who could get merge rights.

**Comment by ghost** [comment]
Sorry -- reinstated as https://github.com/flatpak/flatpak.github.io/pull/354

**Pull request closed without merging**
